### DISCUSSION TITLE
How do i use the "Create a PDF from a single image" code

### DISCUSSION THREAD

**Discussion by 7ory** [dlemstra/Magick.NET#1154] [Help]
I keep getting this error when the PDF is about to get created
![image](https://user-images.githubusercontent.com/50345453/157030402-02689b86-aa59-4734-93e7-c103e6f7fd97.png)

![image](https://user-images.githubusercontent.com/50345453/157030241-c3fc1075-ba99-4b1e-bc91-69e586311116.png)
Now I'm not sure why it's struggling to save the image as a PDF to the destination path please help

**Comment by dlemstra** [comment]
It looks like the application that you are using is not allowed to write to the directory that you specified.

**Comment by 7ory** [comment]
So you're suggesting i change the directory?

**Comment by 7ory** [comment]
Even if i change the directory it still gives me the same error

**Reply by 7ory** [reply]
![image](https://user-images.githubusercontent.com/50345453/157067739-f06956af-86ca-4394-b06e-5ef05f9c3993.png)

Or can you please explain how is the jpg file pulled, as in where is "Snakeware.jpg" stored as it's being called on the using statement, also on the write method where is the "Snakeware.pdf" being saved too why isn't there a path to  specify where it's being saved

**Reply by dlemstra** [reply]
In your code you will need to add paths. I should probably include a path in the examples to make this more clear.

**Reply by 7ory** [reply]
Hi, I finally managed to add a path on where it's going to be saved and there was a problem because windows didn't allow the API to write(create) a new PDF file under the USER directory so I had to change the write/save path to on the C drive so that the API will write directly to the C drive and no permission will be required

**Reply by 7ory** [reply]
I can send a PR adding the path on the master branch if you need me to?

**Reply by dlemstra** [reply]
Thanks for offering to help but earlier today I updated the documentation to add include a path.
